DeepSpeech/fr/train.sh

```diff
@@ -57,7 +62,6 @@ pushd $HOME/ds/
 			--beam_width ${BEAM_WIDTH} \
 			--lm_alpha ${LM_ALPHA} \
 			--lm_beta ${LM_BETA} \
-			${EARLY_STOP_FLAG} \
```

Comment:
is this really wanted?

Reply:
Yes, early stop on export phase makes no sense